### ISSUE TITLE
Failed to install obagent due to there is no nc

### ISSUE THREAD

**Issue by longdafeng** [oceanbase/oceanbase#758]
```

2022-01-13 15:38:21.759  WARN 44 --- [pool-subtask-executor-thread-40,3da594c9516a4c97,3448e43e5d3c] c.a.ocp.core.job.runner.RunnerFactory    : Execute task failed, subtask=SubtaskInstanceEntity{id=72352, name=Check if command exists, state=FAILED, operation=EXECUTE, className=com.alipay.ocp.service.task.business.host.CheckCommandTask, seriesId=23, startTime=2022-01-13T15:38:21.455+08:00, endTime=2022-01-13T15:38:21.757+08:00}, failedMessage=An unknown error has occurred. Cause: command nc not found on host 172.30.62.216. Error message: {1}. Contact the administrator.

```

**Comment by chris-sun-star** [comment]
nc is needed to check obproxy process, the main problem might be there's not enough pre-check before submitting the task, we will add more pre-check.

**Comment by chris-sun-star** [comment]
add precheck when install obproxy, fixed in version 3.1.1-ce-bp1
